Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:7-onbuild
```

Comment:
Personally, I prefer using alpine and installing node myself - not sure what this image is based on though